cinch/roles/nginx/tasks/selinux.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: enable selinux overrides
+  seboolean:
+    name: "{{ item }}"
+    state: yes
```

Comment:
This is by design to achieve compliance with official docs of the module: http://docs.ansible.com/ansible/seboolean_module.html
Using true/false may result in undefined behavior now, or in the future. I would really rather stick to officially supported and documented values.